admin/main.go

```diff
@@ -257,6 +258,20 @@ func init() {
 			EnvVars:     []string{"SERVICE_PORT"},
 			Destination: &adminConfigValues.Port,
 		},
+		&cli.StringFlag{
+			Name:        "log-level",
+			Value:       "info",
```

Comment:
Default value can be a `const`, similarly to the others https://github.com/jmpsec/osctrl/blob/main/admin/main.go#L35